plugins/tiddlywiki/codemirror/license.tid

```diff
@@ -1,15 +1,8 @@
 title: $:/plugins/tiddlywiki/codemirror/license
 
 ~CodeMirror, copyright (c) by Marijn Haverbeke and others
-
```

Comment:
hmm, I didn't think about the rendering problem.  This is what I wanted

```
"""
~CodeMirror, copyright (c) by Marijn Haverbeke and others
Distributed under an MIT license: http://codemirror.net/LICENSE

Copyright (c) 2004-2007, Jeremy Ruston
Copyright (c) 2007-2018, UnaMesa Association
Distributed under an BSD license: https://tiddlywiki.com/#License
"""
```

also see copyright Jeremy is 2018 now :)